app/views/support/users/confirm_destroy.html.erb

```diff
@@ -0,0 +1,23 @@
+<%- title = t('page_titles.confirm_destroy_user') %>
```

Comment:
Page title should be:

"Are you sure you want to delete this user?"

Then the button text:

"Yes, delete this user"

Reply:
fixed and force-pushed

![image](https://user-images.githubusercontent.com/23801/102618780-4d3e5400-4133-11eb-9a16-0a955b0cc97b.png)